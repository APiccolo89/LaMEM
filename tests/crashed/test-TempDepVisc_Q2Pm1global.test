EXEC            = ../bin/LaMEM -ParamFile ./input/TempDepVisc_MG_DAM_NOgridin_benchmark.dat -AddRandomNoiseParticles 0 -restart 0 
NPROCS          = 4
TEST_TYPE       = tolerance_diff_pl,1.0e-3