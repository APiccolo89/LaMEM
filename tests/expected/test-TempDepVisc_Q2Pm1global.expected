 Parsing input file : ./input/TempDepVisc_MG_DAM_NOgridin_benchmark.dat 
# Adding PetscOption: -vpt_element Q2Pm1_global 
# -------------------------------------------------------------------------- 
#                   Lithosphere and Mantle Evolution Model                   
#              Current Revision: 5056 - 2014-01-02 20:44:20		     
#  Modified items: LaMEM_Particles.c, fdstag/JacRes.c, 
#     Compiled: Date: Jan  8 2014 - Time: 13:25:33 - Mode:  	    
# -------------------------------------------------------------------------- 
# LithColDepth[0] = 0 
# LithColDens[0] = 0 
# LithColDens[1] = 0 
# --------------------------------------------------------------------------
# Phase material parameters read from ./input/TempDepVisc_MG_DAM_NOgridin_benchmark.dat: 
#    phase_0 (id=0), VISCOSITY, temp-dep [0,0,0] = 3.200000e-20, 3.000000e+00 1.000000e+00 2.760000e+05 
#    phase_0 (id=0), DENSITY, temperature-dependent [0,1,0] = 2.700000e+03, 0.000000e+00, 0.000000e+00 
#    phase_0 (id=0), ENERGY, constant [0,2,0] = 2.100000e+00 1.050000e+03 4.000000e-07 0.000000e+00 
#    phase_1 (id=1), VISCOSITY, temp-dep [1,0,0] = 3.160000e-26, 3.300000e+00 1.000000e+00 1.900000e+05 
#    phase_1 (id=1), DENSITY, temperature-dependent [1,1,0] = 2.700000e+03, 0.000000e+00, 0.000000e+00 
#    phase_1 (id=1), ENERGY, constant [1,2,0] = 2.500000e+00 1.050000e+03 1.400000e-06 0.000000e+00 
 Stokes solver             : Schur Complement Reduction
 Velocity solver           : Galerkin geometric multigrid
 Element-type used         : Q2Pm1_global;  [Q1P0 Q2Pm1_global Q2Pm1_local Q1Q1 FDSTAG] (change with -vpt_element)
 Total # of cpu's          : 4 
 Total # of velocity dof's : 8019 
 Computing mean material per element using arithmetic averaging from integration points 
 Employing code in a Lagrangian manner 
 Gravity angle with z-axis =  0 
 Approximate Aspect ratio of cells [dx/dz, dy/dz] = [4 4] 
 Number of tracers/cell in [x,y,z]-direction = [3,3,3] 
 NumSurfaceNodes=33  nz=33 
 # Employing nonlinear iterations 
 BC employed: BC.[Left Right; Front Back; Lower Upper]=[6 1; 1 1; 5 0] 
# Initial time step: 0.001 years 
 Input is assumed to be in dimensional units: 
  Characteristic Length 	= 30000 [m] 
  Characteristic Stress 	= 1e+09 [Pa] 
  Characteristic Temperature 	= 623  [Celcius/K] 
  Characteristic Viscosity 	= 1e+24 [Pa.s] 
  Characteristic Time 	= 1e+15 [s] 
 Fine Grid parameters, nel=[4,4,16], cpu=[1,1,4] 
# Finished initializing Stokes matrices : 0.092591 s 
# Not adding random noise to initial particle distribution. 
Initialized particles 
#rank=0, Attributing 1728 particles to elements took 0.0504189 s 
The total # of particles present is 6912 
 You are using the OLD way of setting tracer properties. This piece of code will be REMOVED in a future version of LaMEM - Please change your input scripts!! 
 Reading phase distribution from binary file: input/ParticlesInput_Tbenchmark_2phase.dat   
 Maximum phase-number detected: 1   maxtemp = 1, num_phases = 3 
 Saving initial mesh to file 
 Mesh size: 9 9 33 
 Saved Initial Mesh to file ./InitialMesh/InitialMesh.0.out 
Saved initial particle distribution to directory ./InitialParticles 
The total memory used by integration points is 1.824768e+00 MB
The max memory used by integration points is 4.561920e-01 MB
The min memory used by integration points is 4.561920e-01 MB
#  Interpolating meshes and temperatures took 0.000921011 s 
#  Solving for velocity took 1.3892e+09 s 
#  Forming temperature stiffness matrix mx,my,mz=(9, 9, 33) ...  finished. 
#  Forming temperature stiffness matrixes took 0.013495 s 
#  Solving for temperature took 0.00701189 s 
The total memory used by integration points is 1.824768e+00 MB
The max memory used by integration points is 4.561920e-01 MB
The min memory used by integration points is 4.561920e-01 MB
The total memory used by particles is 1.935360e+00 MB
The max memory used by particles is 4.838400e-01 MB
The min memory used by particles is 4.838400e-01 MB
#  Updating properties @ integration points and particles took 0.0173461 s 
AdvectionMethod = 1 
  Starting particle advection 
  Finished particle advection 
Minimum z-level phase 1 -14687.5 
# Writing output to existing directory Timestep_000000 
# Saved MATLAB output in file Timestep_000000/ThinSheet_Output.0.1000000.out 
# Saved Paraview VTS Velocity output in file ThinSheet_Output_velocity_step000000-mesh.pvts 
# Saved Paraview VTS Temperature output in file ThinSheet_Output_temperature_step000000-mesh.pvts 
# Saved Paraview VTS Surface Topography output in file ThinSheet_Output_Surface_topography_step000000-mesh.pvts 
# Saved Paraview VTS Bottom Topography output in file ThinSheet_Output_Bottom_topography_step000000-mesh.pvts 
# Saved Paraview VTS point-wise quadrature output in file ThinSheet_Output_quadrature_step000000-mesh.pvts 
 Maximum velocity  0   [cm/year]  
 Time = 1e+09, dt=500  [Years]  
# Finished timestep 0 out of 4 in 0.058084 s
  
  
**********************  
 Created a global breakpoint file BreakPoint_Global_VCoord_0.breakpoint 
  # particles = 1728 | rank[0] 
**********************  
  
#  Interpolating meshes and temperatures took 0.00102806 s 
#  Solving for velocity took 1.3892e+09 s 
#  Forming temperature stiffness matrix mx,my,mz=(9, 9, 33) ...  finished. 
#  Forming temperature stiffness matrixes took 0.0137699 s 
#  Solving for temperature took 0.00482297 s 
The total memory used by integration points is 1.824768e+00 MB
The max memory used by integration points is 4.561920e-01 MB
The min memory used by integration points is 4.561920e-01 MB
The total memory used by particles is 1.935360e+00 MB
The max memory used by particles is 4.838400e-01 MB
The min memory used by particles is 4.838400e-01 MB
#  Updating properties @ integration points and particles took 0.01775 s 
AdvectionMethod = 1 
  Starting particle advection 
  Finished particle advection 
Minimum z-level phase 1 -14687.5 
# Writing output to existing directory Timestep_000001 
# Saved MATLAB output in file Timestep_000001/ThinSheet_Output.0.1000001.out 
# Saved Paraview VTS Velocity output in file ThinSheet_Output_velocity_step000001-mesh.pvts 
# Saved Paraview VTS Temperature output in file ThinSheet_Output_temperature_step000001-mesh.pvts 
# Saved Paraview VTS Surface Topography output in file ThinSheet_Output_Surface_topography_step000001-mesh.pvts 
# Saved Paraview VTS Bottom Topography output in file ThinSheet_Output_Bottom_topography_step000001-mesh.pvts 
# Saved Paraview VTS point-wise quadrature output in file ThinSheet_Output_quadrature_step000001-mesh.pvts 
 Maximum velocity  0   [cm/year]  
 Time = 2e+09, dt=500  [Years]  
# Finished timestep 1 out of 4 in 0.056366 s
  
#  Interpolating meshes and temperatures took 0.00095892 s 
#  Solving for velocity took 1.3892e+09 s 
#  Forming temperature stiffness matrix mx,my,mz=(9, 9, 33) ...  finished. 
#  Forming temperature stiffness matrixes took 0.0167401 s 
#  Solving for temperature took 0.00293994 s 
The total memory used by integration points is 1.824768e+00 MB
The max memory used by integration points is 4.561920e-01 MB
The min memory used by integration points is 4.561920e-01 MB
The total memory used by particles is 1.935360e+00 MB
The max memory used by particles is 4.838400e-01 MB
The min memory used by particles is 4.838400e-01 MB
#  Updating properties @ integration points and particles took 0.0154831 s 
AdvectionMethod = 1 
  Starting particle advection 
  Finished particle advection 
Minimum z-level phase 1 -14687.5 
# Writing output to existing directory Timestep_000002 
# Saved MATLAB output in file Timestep_000002/ThinSheet_Output.0.1000002.out 
# Saved Paraview VTS Velocity output in file ThinSheet_Output_velocity_step000002-mesh.pvts 
# Saved Paraview VTS Temperature output in file ThinSheet_Output_temperature_step000002-mesh.pvts 
# Saved Paraview VTS Surface Topography output in file ThinSheet_Output_Surface_topography_step000002-mesh.pvts 
# Saved Paraview VTS Bottom Topography output in file ThinSheet_Output_Bottom_topography_step000002-mesh.pvts 
# Saved Paraview VTS point-wise quadrature output in file ThinSheet_Output_quadrature_step000002-mesh.pvts 
 Maximum velocity  0   [cm/year]  
 Time = 3e+09, dt=500  [Years]  
# Finished timestep 2 out of 4 in 0.055881 s
  
#  Interpolating meshes and temperatures took 0.000962019 s 
#  Forming FEM stiffness matrix mx,my,mz=(9, 9, 33) ...   finished. [0.113439 s]
 minDiagonalRatio=1 [Critical = 0.55; change with -CriticalDiagonalRatio]  
#  Forming stiffness matrixes took 0.115375 s 
SCR: Viscosity structure info: min(0)=1.000000e-05 : max(0)=1.000000e-05 : contrast=1.000000e+00 
  Residual norms for scr_ solve.
  0 KSP Residual norm 2.193826997650e+03 
  1 KSP Residual norm 3.502772051288e-04 
Linear solve converged due to CONVERGED_RTOL iterations 1
# Time: p solve = 1.31e-01 
# Time: u solve = 2.47e-02 
#  Finished Stokes solve after 0.4697 s 
------------------------------------------
Stokes solver summary: 
  divergence: 
    Div_min = 2.158138766027e-07 
    Div_max = 2.527019821738e-04 
    |Div|_2 = 2.797455251043e-03 
    |Div|_1 = 7.375758624537e-02 
  momentum: 
    mRes_min = 2.102205558563e-15 
    mRes_max = 2.820130272569e-08 
    |mRes|_2 = 1.902008475898e-07 
    |mRes|_1 = 6.317857011222e-06 
  velocity: 
    u_min   = -1.000000000626e+00 
    u_max   = 1.007401430891e+00 
    |u|_2   = 4.279074065147e+01 
    |u|_1   = 2.655411654261e+03 
  pressure: 
    p_min   = -8.100040924520e-01 
    p_max   = 2.307635136060e-05 
    |p|_2   = 1.295999713005e+01 
    |p|_1   = 2.073652575210e+02 
------------------------------------------
The total memory used by integration points is 1.824768e+00 MB
The max memory used by integration points is 4.561920e-01 MB
The min memory used by integration points is 4.561920e-01 MB
Timestep 3, NONLINEAR ITERATION 0 [from 25], accuracy level=0.0005, current velocity error: 1  
 
#  Forming FEM stiffness matrix mx,my,mz=(9, 9, 33) ...   finished. [0.114031 s]
 minDiagonalRatio=1 [Critical = 0.55; change with -CriticalDiagonalRatio]  
#  Forming stiffness matrixes took 0.115082 s 
SCR: Viscosity structure info: min(12)=2.027106e+00 : max(1008)=2.400718e+04 : contrast=1.184308e+04 
  Residual norms for scr_ solve.
  0 KSP Residual norm 8.826413779551e-02 
  1 KSP Residual norm 1.060592323177e-03 
  2 KSP Residual norm 8.033820739313e-04 
  3 KSP Residual norm 2.830806162151e-04 
  4 KSP Residual norm 1.105464526343e-04 
  5 KSP Residual norm 2.886800964455e-05 
  6 KSP Residual norm 1.044740935699e-05 
  7 KSP Residual norm 5.654375283932e-06 
  8 KSP Residual norm 2.655001657875e-06 
  9 KSP Residual norm 6.970388966547e-07 
 10 KSP Residual norm 2.745983683729e-07 
 11 KSP Residual norm 1.082929886179e-07 
 12 KSP Residual norm 6.207628780498e-08 
Linear solve converged due to CONVERGED_RTOL iterations 12
# Time: p solve = 3.67e+00 
# Time: u solve = 2.12e-01 
#  Finished Stokes solve after 4.34338 s 
------------------------------------------
Stokes solver summary: 
  divergence: 
    Div_min = 4.435709612116e-14 
    Div_max = 2.306915366200e-08 
    |Div|_2 = 8.147836440162e-08 
    |Div|_1 = 1.406824366392e-06 
  momentum: 
    mRes_min = 3.016480836816e-15 
    mRes_max = 7.640609317155e-06 
    |mRes|_2 = 3.432426222577e-05 
    |mRes|_1 = 5.736678608810e-04 
  velocity: 
    u_min   = -1.000000000486e+00 
    u_max   = 1.003817453765e+00 
    |u|_2   = 4.301459541745e+01 
    |u|_1   = 2.672039481923e+03 
  pressure: 
    p_min   = -1.676982151276e+03 
    p_max   = 4.784059552111e+04 
    |p|_2   = 1.912661411487e+05 
    |p|_1   = 9.001741830368e+05 
------------------------------------------
The total memory used by integration points is 1.824768e+00 MB
The max memory used by integration points is 4.561920e-01 MB
The min memory used by integration points is 4.561920e-01 MB
Timestep 3, NONLINEAR ITERATION 1 [from 25], accuracy level=0.0005, current velocity error: 0.0114099  
 
#  Forming FEM stiffness matrix mx,my,mz=(9, 9, 33) ...   finished. [0.114676 s]
 minDiagonalRatio=1 [Critical = 0.55; change with -CriticalDiagonalRatio]  
#  Forming stiffness matrixes took 0.115865 s 
SCR: Viscosity structure info: min(52)=2.017090e+00 : max(1008)=2.386527e+04 : contrast=1.183154e+04 
  Residual norms for scr_ solve.
  0 KSP Residual norm 7.715052649823e-04 
  1 KSP Residual norm 5.474787681093e-04 
  2 KSP Residual norm 2.671021778561e-04 
  3 KSP Residual norm 8.162713967913e-05 
  4 KSP Residual norm 3.328840881220e-05 
  5 KSP Residual norm 1.280702793750e-05 
  6 KSP Residual norm 4.384886320993e-06 
  7 KSP Residual norm 2.545264626183e-06 
  8 KSP Residual norm 8.385457332436e-07 
  9 KSP Residual norm 3.316936303239e-07 
 10 KSP Residual norm 1.230265784820e-07 
 11 KSP Residual norm 4.866990283175e-08 
Linear solve converged due to CONVERGED_RTOL iterations 11
# Time: p solve = 3.41e+00 
# Time: u solve = 1.78e-01 
#  Finished Stokes solve after 4.05734 s 
------------------------------------------
Stokes solver summary: 
  divergence: 
    Div_min = 8.836551282365e-14 
    Div_max = 1.331685671544e-08 
    |Div|_2 = 6.291250615490e-08 
    |Div|_1 = 1.233288278554e-06 
  momentum: 
    mRes_min = 1.956127427532e-14 
    mRes_max = 1.697553023969e-05 
    |mRes|_2 = 1.142927346541e-04 
    |mRes|_1 = 2.153679537897e-03 
  velocity: 
    u_min   = -1.000000007202e+00 
    u_max   = 1.001836310606e+00 
    |u|_2   = 4.302229194338e+01 
    |u|_1   = 2.672320632239e+03 
  pressure: 
    p_min   = -7.690696917659e+02 
    p_max   = 4.766728935497e+04 
    |p|_2   = 1.911525245818e+05 
    |p|_1   = 8.842627696040e+05 
------------------------------------------
The total memory used by integration points is 1.824768e+00 MB
The max memory used by integration points is 4.561920e-01 MB
The min memory used by integration points is 4.561920e-01 MB
Timestep 3, NONLINEAR ITERATION 2 [from 25], accuracy level=0.0005, current velocity error: 0.00185234  
 
#  Forming FEM stiffness matrix mx,my,mz=(9, 9, 33) ...   finished. [0.10934 s]
 minDiagonalRatio=1 [Critical = 0.55; change with -CriticalDiagonalRatio]  
#  Forming stiffness matrixes took 0.109901 s 
SCR: Viscosity structure info: min(48)=2.030037e+00 : max(1008)=2.380509e+04 : contrast=1.172644e+04 
  Residual norms for scr_ solve.
  0 KSP Residual norm 1.772902189739e-04 
  1 KSP Residual norm 9.011557197819e-05 
  2 KSP Residual norm 4.168610881526e-05 
  3 KSP Residual norm 1.193261338650e-05 
  4 KSP Residual norm 5.666189392703e-06 
  5 KSP Residual norm 2.175934387467e-06 
  6 KSP Residual norm 7.680230481406e-07 
  7 KSP Residual norm 4.253739663181e-07 
  8 KSP Residual norm 1.883137153282e-07 
  9 KSP Residual norm 4.411324670536e-08 
Linear solve converged due to CONVERGED_RTOL iterations 9
# Time: p solve = 2.69e+00 
# Time: u solve = 1.53e-01 
#  Finished Stokes solve after 3.30334 s 
------------------------------------------
Stokes solver summary: 
  divergence: 
    Div_min = 3.054159030715e-13 
    Div_max = 1.687752920754e-08 
    |Div|_2 = 7.616135581467e-08 
    |Div|_1 = 1.509804555882e-06 
  momentum: 
    mRes_min = 2.725202598467e-14 
    mRes_max = 1.970261416197e-05 
    |mRes|_2 = 9.505621017080e-05 
    |mRes|_1 = 1.721455789782e-03 
  velocity: 
    u_min   = -1.000000001908e+00 
    u_max   = 1.000996662532e+00 
    |u|_2   = 4.302592631767e+01 
    |u|_1   = 2.672624510335e+03 
  pressure: 
    p_min   = -4.001422980322e+02 
    p_max   = 4.757707786188e+04 
    |p|_2   = 1.910249555633e+05 
    |p|_1   = 8.764230073967e+05 
------------------------------------------
The total memory used by integration points is 1.824768e+00 MB
The max memory used by integration points is 4.561920e-01 MB
The min memory used by integration points is 4.561920e-01 MB
Timestep 3, NONLINEAR ITERATION 3 [from 25], accuracy level=0.0005, current velocity error: 0.00106167  
 
#  Forming FEM stiffness matrix mx,my,mz=(9, 9, 33) ...   finished. [0.124941 s]
 minDiagonalRatio=1 [Critical = 0.55; change with -CriticalDiagonalRatio]  
#  Forming stiffness matrixes took 0.125988 s 
SCR: Viscosity structure info: min(32)=2.037145e+00 : max(1008)=2.377216e+04 : contrast=1.166935e+04 
  Residual norms for scr_ solve.
  0 KSP Residual norm 9.146941536811e-05 
  1 KSP Residual norm 4.132009289071e-05 
  2 KSP Residual norm 2.060532186672e-05 
  3 KSP Residual norm 7.512539848713e-06 
  4 KSP Residual norm 3.618129184082e-06 
  5 KSP Residual norm 9.544583732059e-07 
  6 KSP Residual norm 3.316997049117e-07 
  7 KSP Residual norm 1.873750288121e-07 
  8 KSP Residual norm 7.789212288278e-08 
Linear solve converged due to CONVERGED_RTOL iterations 8
# Time: p solve = 2.66e+00 
# Time: u solve = 1.67e-01 
#  Finished Stokes solve after 3.30739 s 
------------------------------------------
Stokes solver summary: 
  divergence: 
    Div_min = 1.282700074628e-13 
    Div_max = 1.804769510401e-08 
    |Div|_2 = 9.028757928877e-08 
    |Div|_1 = 1.647757462123e-06 
  momentum: 
    mRes_min = 2.238532997883e-14 
    mRes_max = 5.060434246523e-06 
    |mRes|_2 = 2.684455434760e-05 
    |mRes|_1 = 5.545030964301e-04 
  velocity: 
    u_min   = -1.000000000631e+00 
    u_max   = 1.000580757775e+00 
    |u|_2   = 4.302777820976e+01 
    |u|_1   = 2.672803816348e+03 
  pressure: 
    p_min   = -2.250317223386e+02 
    p_max   = 4.752441169888e+04 
    |p|_2   = 1.909366742017e+05 
    |p|_1   = 8.722573090584e+05 
------------------------------------------
The total memory used by integration points is 1.824768e+00 MB
The max memory used by integration points is 4.561920e-01 MB
The min memory used by integration points is 4.561920e-01 MB
Timestep 3, NONLINEAR ITERATION 4 [from 25], accuracy level=0.0005, current velocity error: 0.0006132  
 
#  Forming FEM stiffness matrix mx,my,mz=(9, 9, 33) ...   finished. [0.119085 s]
 minDiagonalRatio=1 [Critical = 0.55; change with -CriticalDiagonalRatio]  
#  Forming stiffness matrixes took 0.120028 s 
SCR: Viscosity structure info: min(0)=2.041339e+00 : max(1008)=2.375233e+04 : contrast=1.163566e+04 
  Residual norms for scr_ solve.
  0 KSP Residual norm 4.987499912059e-05 
  1 KSP Residual norm 2.268127654015e-05 
  2 KSP Residual norm 1.131517177478e-05 
  3 KSP Residual norm 4.087849134888e-06 
  4 KSP Residual norm 1.768915697695e-06 
  5 KSP Residual norm 3.971347649484e-07 
  6 KSP Residual norm 1.493953502245e-07 
  7 KSP Residual norm 9.041693202433e-08 
Linear solve converged due to CONVERGED_RTOL iterations 7
# Time: p solve = 2.10e+00 
# Time: u solve = 1.21e-01 
#  Finished Stokes solve after 2.69973 s 
------------------------------------------
Stokes solver summary: 
  divergence: 
    Div_min = 4.605560724458e-14 
    Div_max = 2.662725165283e-08 
    |Div|_2 = 9.737082088587e-08 
    |Div|_1 = 1.766374966410e-06 
  momentum: 
    mRes_min = 7.316048073608e-15 
    mRes_max = 8.768309331119e-06 
    |mRes|_2 = 5.206246521365e-05 
    |mRes|_1 = 9.653967646018e-04 
  velocity: 
    u_min   = -1.000000001639e+00 
    u_max   = 1.000366498573e+00 
    |u|_2   = 4.302873674887e+01 
    |u|_1   = 2.672896157971e+03 
  pressure: 
    p_min   = -1.388593331134e+02 
    p_max   = 4.749233268073e+04 
    |p|_2   = 1.908791079894e+05 
    |p|_1   = 8.699418666074e+05 
------------------------------------------
The total memory used by integration points is 1.824768e+00 MB
The max memory used by integration points is 4.561920e-01 MB
The min memory used by integration points is 4.561920e-01 MB
Timestep 3, NONLINEAR ITERATION 5 [from 25], accuracy level=0.0005, current velocity error: 0.000354327  
 
#  Solving for velocity took 2.8309 s 
#  Forming temperature stiffness matrix mx,my,mz=(9, 9, 33) ...  finished. 
#  Forming temperature stiffness matrixes took 0.016222 s 
#  Solving for temperature took 0.00291395 s 
The total memory used by integration points is 1.824768e+00 MB
The max memory used by integration points is 4.561920e-01 MB
The min memory used by integration points is 4.561920e-01 MB
The total memory used by particles is 1.935360e+00 MB
The max memory used by particles is 4.838400e-01 MB
The min memory used by particles is 4.838400e-01 MB
#  Updating properties @ integration points and particles took 0.0173349 s 
AdvectionMethod = 1 
  Starting particle advection 
  Finished particle advection 
Minimum z-level phase 1 -14687.2 
# Writing output to existing directory Timestep_000003 
# Saved MATLAB output in file Timestep_000003/ThinSheet_Output.0.1000003.out 
# Saved Paraview VTS Velocity output in file ThinSheet_Output_velocity_step000003-mesh.pvts 
# Saved Paraview VTS Temperature output in file ThinSheet_Output_temperature_step000003-mesh.pvts 
# Saved Paraview VTS Surface Topography output in file ThinSheet_Output_Surface_topography_step000003-mesh.pvts 
# Saved Paraview VTS Bottom Topography output in file ThinSheet_Output_Bottom_topography_step000003-mesh.pvts 
# Saved Paraview VTS point-wise quadrature output in file ThinSheet_Output_quadrature_step000003-mesh.pvts 
 Maximum velocity  0.0947075   [cm/year]  
 Time = 3e+09, dt=500  [Years]  
# Finished timestep 3 out of 4 in 19.003 s
  
# Total time required: 19.3659 s 
#PETSc Option Table entries:
-AddRandomNoiseParticles 0
-ParamFile ./input/TempDepVisc_MG_DAM_NOgridin_benchmark.dat
-options_left
-restart 0
-scr_ksp_converged_reason
-scr_ksp_max_it 1000
-scr_ksp_monitor
-scr_ksp_rtol 1.0e-6
-scr_ksp_type fgmres
-scr_pc_type sor
-vpt_element Q2Pm1_global
-vs_ksp_max_it 1000
-vs_ksp_rtol 1e-8
-vs_ksp_type fgmres
-vs_mg_coarse_ksp_type preonly
-vs_mg_coarse_pc_redundant_number 2
-vs_mg_coarse_pc_type redundant
-vs_mg_coarse_redundant_pc_factor_mat_solver_package mumps
-vs_mg_levels_ksp_max_it 4
-vs_mg_levels_ksp_type gmres
-vs_mg_levels_pc_fieldsplit_block_size 3
-vs_mg_levels_pc_fieldsplit_type ADDITIVE
-vs_mg_levels_pc_type fieldsplit
-vs_pc_mg_cycle_type v
-vs_pc_mg_galerkin
-vs_pc_mg_levels 2
-vs_pc_mg_type multiplicative
-vs_pc_type mg
#End of PETSc Option Table entries
There are no unused options.
